src/components/CardAdministration.vue

```diff
@@ -200,6 +203,18 @@ const toast = useToast();
 
 const { mutateAsync: deleteAdministration } = useDeleteAdministrationMutation();
 
+const status = computed(() => {
+  const now = new Date();
+  const dateStart = new Date(props.dates.start);
+  const dateClosed = new Date(props.dates.end);
+
+  if (now > dateClosed) return 'CLOSED';
+  if (now >= dateStart && now <= dateClosed) return 'IN PROGRESS';
+  return 'OPEN';
+});
+
+const statusClass = computed(() => status.value.toLowerCase().replace(' ', '-'));
```

Comment:
Is there a good reason to use separate computed values here?

We could store the status in a variable and then transform it.

```suggestion
const status = computed(() => {
  const now = new Date();
  const dateStart = new Date(props.dates.start);
  const dateClosed = new Date(props.dates.end);
  
  let untransformedStatus = 'OPEN'

  if (now > dateClosed) untransformedStatus = 'CLOSED';
  if (now >= dateStart && now <= dateClosed) untransformedStatus =  'IN PROGRESS';
  
  return untransformedStatus.toLowerCase().replace(' ', '-')
});

const statusClass = computed(() => status.value.toLowerCase().replace(' ', '-'));
```

Reply:
Also apologies for being pedantic but could we name the variable holding the computed value to something like "administrationStatus"? "status" is too vague and "statusClass" may be confusing since we have "Class" as an org type.

Reply:
It makes perfect sense too. I sent the improvement.